.storybook/preview.ts

```diff
@@ -61,6 +64,41 @@ const preview: Preview = {
       defaultTheme: "light",
       attributeName: "is-dark-theme",
     }),
+    () => {
```

Comment:
何してるかちょっとメモっときたいかも。
こうですかね？
```suggestion
    // テーマ用データ属性を監視してテーマを追従する
    () => {
```